ONSdigital/dp-frontend-search-controller
# Feature/data aggregation page one form

### What
This [ticket](https://trello.com/c/IGM2XaH8) was to refactor the aggregation pages into one form.
Including removing unneeded filter submit buttons when js is disabled, just having a single 'apply filters' at the bottom.

### How to review

- Pull this branch and run app make debug ENABLE_REWORKED_DATA_AGGREGATION_PAGES=true
- Run dp-design-system
- Port forward the api router to sandbox dp ssh sandbox web 1 -p 23200:localhost:10800

Pages:

- https://dp.aws.onsdigital.uk/allmethodologies
- https://dp.aws.onsdigital.uk/datalist
- https://dp.aws.onsdigital.uk/timeseriestool
- https://dp.aws.onsdigital.uk/publications
- https://dp.aws.onsdigital.uk/staticlist
- https://dp.aws.onsdigital.uk/topicspecificmethodology
- https://dp.aws.onsdigital.uk/alladhocs
- https://dp.aws.onsdigital.uk/publishedrequests

Check filter functionality (including sort) still works including non-js journey.
Inputs retain values.
Sense check.

### Who can review

Frontend
